docs/tutorial/02-hello-world.mdx

```diff
@@ -26,42 +26,59 @@ A smart contract is a program that verifies and executes the performance of a co
+Before you get started with Flow, you need to understand how accounts and transactions are modeled on Flow and in Cadence.
 
 ## Accounts and Transactions
 
 ---
 
 Like most other blockchains, the programming model in Flow is centered around accounts and transactions.
-All persistent state is stored in [accounts](/cadence/language/accounts).
+All state that persists permanentely is stored in [accounts](/cadence/language/accounts)
+and there is only one type of account (all accounts can be for any number of users, smart contracts, and data).
+
 The interfaces to this state (the ways to interact with it) are also stored in accounts.
-All code execution takes place within transactions, which are blocks of code that are submitted by external users to interact with the persistent state,
+All code execution takes place within [transactions](/cadence/language/transactions), which are blocks of code
+that are authorized and submitted by external users to interact with the persistent state,
 which includes directly modifying account storage.
 
-Each user has an account controlled by one or more private keys.
-This means that support for accounts/wallets with [multiple controllers](https://www.coindesk.com/what-is-a-multisignature-crypto-wallet) is built into the protocol by default.
+### What is an account?
+
+Each user has an account controlled by one or more private keys with configurable weight.
+This means that support for accounts/wallets with [multiple controllers](https://www.coindesk.com/what-is-a-multisignature-crypto-wallet)
+is built into the protocol by default.
 
 An account is divided into two main areas:
 
 1. The first area is the [contract area](/cadence/language/accounts).
    This is the area that stores any number smart contracts that contain type definitions, fields,
    and functions that relate to a common functionality.
    This area also holds contract interfaces, which are basically program guidelines that other contracts can import and implement.
-   This area cannot be directly read from in a transaction, unless the transaction is just reading the code deployed to an account.
+   This area cannot be directly accessed in a transaction unless the transaction is just reading a copy of the code deployed to an account.
    The owner of an account can directly add, remove, or update/overwrite contracts that are stored in it.
 
-2. The second area is the account filesystem. This area is where an account stores the objects
-   that they own and the capabilities for controlling how these objects are accessed.
+2. The second area is the account storage. This area is where an account stores the objects
+   that they own and the capabilities for controlling how these objects can be accessed.
+
+### Account Storage
+
+The account storage functions similar to a traditional filesystem;
+Data is stored in paths, links can be created between paths that define different permissions and functionality,
```

Comment:
What does between paths mean? I'm not sure I understand (personal question) 

May be confusing for the reader?